File: 000-099/000-009/002.c
#include <stdio.h>
int main(){
    int a1=1,a2=2,a3,sum=2;
    for(;a3<=4000000;){
        a3=a1+a2;
        if (!(a3%2)) sum+=a3;
        a1=a2;
        a2=a3;
    }
    printf("answer : %d\n", sum);
    return 0;
}

File: 000-099/000-009/002.hs
fib :: Int -> Int
fib 0 = 0
fib 1 = 1
fib n = fib (n-1) + fib (n-2)

fibsUnder4million = takeWhile (<4000000) $ map fib [1..]
answer = sum $ filter even fibsUnder4million

main = print (answer)
